lib/mshoplib/src/MShop/Service/Provider/Decorator/Supplier.php

```diff
@@ -179,7 +179,7 @@ public function setConfigFE( \Aimeos\MShop\Order\Item\Base\Service\Iface $orderS
 		if( ( $code = $attributes['supplier.code'] ) != '' )
 		{
 			// add short address as attribute for summary page / customer email
-			$attributes['supplier.address'] = $this->feConfig['supplier.code']['short'][$code];
+			$attributes['supplier.address'] = $this->feConfig['supplier.code']['default'][$code];
```

Comment:
I think with this change, it allow us to choose the address format, but i don't know how to call config from this file.

$attributes['supplier.address'] = $this->feConfig['supplier.code'][  <code>$this->config('client/format','default')</code>  ][$code];

Reply:
`$this->getConfigValue( 'supplier.addressformat' )` will do the job if you add it to the `$beConfig`